### ISSUE TITLE
can't find crate for `std`

### ISSUE THREAD

**Issue by nkrackow** [DerFetzer/colorlight-litex#1]
Hey sorry if this is an obvious thing and I just don't know what I'm doing..
And thanks for making this available! 

Unfortunately I get
> error[E0463]: can't find crate for `std`
>   |
>   = note: the `riscv32i-unknown-none-elf` target may not be installed
> 
> error: aborting due to previous error
> 
> For more information about this error, try `rustc --explain E0463`.
> error: could not compile `log`

when trying to compile the eth_demo. I tried using rustc 1.48.0-nightly and 1.50.0, but that doesn't make a difference. 
[The icebreaker example](https://github.com/icebreaker-fpga/icebreaker-litex-examples) works just fine.

**Comment by DerFetzer** [comment]
Hey,
thanks for your problem report.

Something does not seem quite right with your setup. I updated the target sometime to `riscv32imac-unknown-none-elf` and this is also in the `cargo` config file. So I don't know, why your error message says something about `riscv32i-unknown-none-elf`.

Make sure you are in the `eth_demo` folder and do not override the target somehow.
Maybe you have to install the `riscv32imac-unknown-none-elf` target. I corrected that in the readme as well.
The project compiles for me with `rustc 1.48.0-nightly (f3c923a13 2020-09-17)`.

**Comment by nkrackow** [comment]
Hi, thaks for the reply!

Yeah that was the only thing I changed bc there was a mismatch to the instruction and it did not seem right with the imac (the vexriscV is just i right?).
However, I now get the same with imac(ofc i installed that target first).. 

If I find some time over the next days I'll try to adapt the Icebreaker example to Colorlight and get an LED blinking. Maybe I'll find the problem along the way :)

**Comment by DerFetzer** [comment]
As I am using the `IMAC` version of `VexRiscV` this should work with `riscv32imac-unknown-none-elf`.

I had a similar problem at first so I added the unstable feature `host_dep` to the cargo config. Can you check if this is correctly enabled? I'll try sometime if this is still needed on latest nightly or if it already got fixed.

**Comment by nkrackow** [comment]
> As I am using the IMAC version of VexRiscV this should work with riscv32imac-unknown-none-elf.

Really?? Wow cool, so we've got a proper computer in there :D

I figured it out.. I typed `rustup default nightly-2020-09-17` which actually gives you `rustc 1.48.0-nightly (285fc7d70 2020-09-16)`. Using `rustup default nightly-2020-09-18` gives you the exact `rustc 1.48.0-nightly (f3c923a13 2020-09-17)` that works!
Wow that is very version specific! :D Cool, I think I can start playing over the weekend now! :)

Btw one small thing you might want to mention is that ppl need to install [ecpprog](https://github.com/gregdavill/ecpprog) too.
